staging/src/k8s.io/code-generator/cmd/validation-gen/validators/openapi.go

```diff
@@ -64,26 +69,40 @@ func (openAPIDeclarativeValidator) ExtractValidations(t *types.Type, comments []
 }
 
 func (openAPIDeclarativeValidator) Docs() []TagDoc {
-	return []TagDoc{{
-		Tag:         formatTagName,
-		Description: "Indicates that a string field has a particular format.",
-		Contexts:    []TagContext{TagContextType, TagContextField},
-		Payloads: []TagPayloadDoc{{
-			Description: "ip",
-			Docs:        "This field holds an IP address value, either IPv4 or IPv6.",
-		}, {
-			Description: "dns-label",
-			Docs:        "This field holds a DNS label value.",
-		}},
-	}, {
-		Tag:         maxLengthTagName,
-		Description: "Indicates that a string field has a limit on its length.",
-		Contexts:    []TagContext{TagContextType, TagContextField},
-		Payloads: []TagPayloadDoc{{
-			Description: "<non-negative integer>",
-			Docs:        "This field must be no more than X characters long.",
-		}},
-	}}
+	return []TagDoc{
```

Comment:
Why did you un-cuddle these?  Less vertical space was the purpose :)

Reply:
Oh.. I didn't know. Changed it back